js/src/ui/Portal/portal.js

```diff
@@ -59,21 +59,26 @@ export default class Portal extends Component {
     const { children, className } = this.props;
 
     const classes = [ styles.overlay, className ];
+    const backClasses = [ styles.backOverlay ];
 
     if (expanded) {
       classes.push(styles.expanded);
+      backClasses.push(styles.expanded);
     }
 
     return (
       <ReactPortal isOpened onClose={ this.handleClose }>
-        <div
-          className={ classes.join(' ') }
-          onKeyDown={ this.handleKeyDown }
-        >
-          <ParityBackground className={ styles.parityBackground } />
-
-          { this.renderCloseIcon() }
-          { children }
+        <div className={ backClasses.join(' ') } onClick={ this.handleClose }>
+          <div
+            className={ classes.join(' ') }
+            onKeyDown={ this.handleKeyDown }
+            onClick={ this.stopEvent }
```

Comment:
onClick before onKeyDown, but just my OCD kicking in. (Doesn't make a huge difference here with the small amount of attributes but prefer following a pattern everywhere for the sake of consistency)